index.html

```diff
@@ -2,9 +2,16 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8" />
-    <link rel="icon" type="image/svg+xml" href="/icon.jpg" />
+    <link rel="icon" type="image/svg+xml" href="/icon.png" />
```

Comment:
Double check the note in Trello. The "type" attribute should be different.

Reply:
changed boss